src/main/scala/mesosphere/chaos/http/HttpConf.scala

```diff
@@ -5,6 +5,11 @@ import java.net.URL
 
 trait HttpConf extends ScallopConf {
 
+  lazy val httpDisable = opt[Boolean]("http_disable",
```

Comment:
I agree with you and I started out with that version.

The only reason I changed it to `--http-disable` was that I wanted all http options to close to each other in `--help`. `--help` displays all options in lexicographical order.

Unfortunately, I cannot clarify with you what's more important to you, so I leave it like it is.